lib/timescale/migration.ex

```diff
@@ -67,9 +67,8 @@ defmodule Timescale.Migration do
   defp opts_to_sql_arg([]), do: ""
 
   defp opts_to_sql_arg(opts) do
-    ", " <>
-      Enum.map(opts, fn {k, v} ->
-        "#{k} => #{v}"
-      end)
+    Enum.map_join(opts, ", ", fn {k, v} ->
+      "#{k} => #{v}"
+    end)
```

Comment:
I'd like to figure out a clean way to get test coverage for the `migration.ex` file 🤔